.github/workflows/testsAndReviews.yml

```diff
@@ -1,68 +1,20 @@
 name: Node.js Continuous Integration
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow pull request events but only for "main" and "dev" branch
   pull_request:
     branches: [ "dev", "main" ]
 
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-#jobs that will be ran on every PR or push
 jobs:
-#job 1 - automatically set reviewers
-  auto_set_reviewers: 
```

Comment:
Since the auto set reviewers part has been taken out of the code, along with the array of github users to assign PR's to, do we still have the logic to assign reviewers?